### DISCUSSION TITLE
Smart Nora Google Play store App

### DISCUSSION THREAD

**Discussion by rgment** [andrei-tatar/node-red-contrib-smartnora#302] [General]
Smart Nora Google App is missing from the play store.
I have the app installed on my old Pixel 6 Phone, install about 6 months ago, and still working.
Upgrade to a new Pixel phone but can't find the app in the Google play store to install it.
Thanks for you help in advance.

**Comment by andrei-tatar** [comment]
@rgment there was never a Google play store app. It's a PWA that can be installed on phones.

If you go at https://smart-nora.eu/ the browser (if supported by the phone) will prompt you to install the app

**Reply by rgment** [reply]
Thank you very much andrei-tatar. It worked Flawlessly, the App is now installed.
I guess I was wrong about installing it from the Google play store.
